canonical-web-and-design/snapcraft.io
# Refactor to use canonicalwebteam.store_api

Refactor to use the Python package: https://github.com/canonical-web-and-design/canonicalwebteam.store-api

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/
- Ensure that the site works and you are able to search snaps and get snap details
